Containerfile

```diff
@@ -10,6 +10,7 @@ RUN set -exo pipefail \
         --disablerepo=* \
         --enablerepo=fedora,updates \
         gcc \
+        krb5-devel \
```

Comment:
sure

Reply:
This is needed here, otherwise build container step will failed, https://console.redhat.com/preview/application-pipeline/workspaces/exd-sp-rhel-wf/applications/retasc/pipelineruns/retasc-on-pull-request-l9t4p/logs?task=build-container
It is at the same location as https://github.com/release-engineering/resultsdb/blob/480b63f262c7bc80e26c30fba42f2f7757a74917/Dockerfile#L13

Reply:
I understand it is needed, but you are likely linking against krb5 libraries when poetry is building some Python packages, so `krb5-libs` must be installed in the final container image (see it in the resultsdb Dockerfile) otherwise the resulting container will have missing library dependencies.

Reply:
I see what you mean now, I misunderstood your original comment. Fixing it now.